doc/api/dgram.md

```diff
@@ -283,6 +283,9 @@ not work because the packet will get silently dropped without informing the
 source that the data did not reach its intended recipient.
 
 ### socket.setBroadcast(flag)
+<!-- YAML
+added: v0.6.9
```

Comment:
I'm fairly sure it goes all the way back to v0.2.0.  It was certainly in v0.4.x because I wrote the libuv replacement during the v0.5.x development cycle.